index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>apple-clone</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+</head>
+<body>
+    <main>
+        <div class = "main-image">
+            <nav>
+                <ul>
+                    <li>
+                        <button class="apple-icon">
+                            <i class="fa fa-apple" aria-hidden="true"></i>
+                        </button>
+                    </li>
+                    <li>
+                        <button>Store</button>
+                    </li>
+                    <li>
+                        <button>Mac</button>
+                    </li>
+                    <li>
+                        <button>iPod</button>
+                    </li>
+                    <li>
+                        <button>iPhone</button>
+                    </li>
+                    <li>
+                        <button>iPad</button>
+                    </li>
+                    <li>
+                        <button>iTunes</button>
+                    </li>
+                    <li>
+                        <button>Support</button>
+                    </li>
+                    <li class="right button">
+                        <i class="fa fa-search custom-user"  aria-hidden="true"></i><input type="search" name="search">
+                    </li>
+                </ul>
+            </nav>
+            <div>
+                <h1>What will your verse be?</h1>
+                <p class="sub-heading">Everyone has something to share.See how people are using
+                    <br>iPad in their unique ways to add to the world's story
+                    <br>
+                    Watch the "Your verse" film and explore the stories
+                    <i class="fa fa-play-circle-o" aria-hidden="true"></i>
+                </p>
+            </div>
+        </div>
```

Comment:
- [ ] These elements are links. If you check the original page you can see there is no need to write the text, you can just copy the images and position them. Choose an option, but make sure they are clickable and that the look is the same (they should have more space between them)